scripts/run_tests

```diff
@@ -4,13 +4,19 @@ readonly SCRIPTS_DIR="$(dirname "$(readlink -f "$0")")"
 # shellcheck source=scripts/common
 source "$SCRIPTS_DIR/common"
 
-# For each Rust workspace, first run tests, then run clippy, turning warnings into errors.
-# See https://github.com/rust-lang/rust-clippy.
+# For each Rust workspace, first run tests, then run doc tests, then run clippy (turning warnings
+# into errors).
+#
+# See:
+# - https://doc.rust-lang.org/cargo/commands/cargo-test.html
+# - https://github.com/rust-lang/rust-clippy.
 
 cargo test --all-targets --manifest-path=./sdk/rust/Cargo.toml
```

Comment:
(existing) I think this is missing `oak/server/rust` code, also for `format`/`check_formatting`, you could add it to this PR?

Reply:
Yeah, it would be nice to auto-find Rust workspace files; cf. https://github.com/project-oak/oak/pull/466#discussion_r366403818

Reply:
Done now (auto-discovery, but in the `runner` script only. Good news is that it has not auto-discovered any other workspace file so far :) )